src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+    public static String parseDate(String str) {
```

Comment:
I fear some might find this misleading as `parseDate` could be interpreted as parsing the string to get a `LocalDate` object. Maybe consider `formatDateString(String)`?

Reply:
i think that's a good idea!